src/ast.c

```diff
@@ -1140,7 +1146,7 @@ JL_DLLEXPORT jl_value_t *jl_expand_with_loc_warn(jl_value_t *expr, jl_module_t *
     JL_TIMING(LOWERING);
     JL_GC_PUSH1(&expr);
     expr = jl_copy_ast(expr);
-    expr = jl_expand_macros(expr, inmodule, NULL, 0);
+    expr = jl_expand_macros(expr, inmodule, NULL, 0, jl_world_counter);
```

Comment:
```suggestion
    expr = jl_expand_macros(expr, inmodule, NULL, 0,  ~(size_t)0);
```

Reply:
Looks like an unintented whitespace snuck in?
```suggestion
    expr = jl_expand_macros(expr, inmodule, NULL, 0, ~(size_t)0);
```